payment/config.py

```diff
@@ -0,0 +1,9 @@
+import os
+
+
+STELLAR_HORIZON_URL = 'https://horizon-testnet.stellar.org/'
+STELLAR_NETWORK = 'TESTNET'
+STELLAR_KIN_ISSUER_ADDRESS = 'GCKG5WGBIJP74UDNRIRDFGENNIH5Y3KBI5IHREFAJKV4MQXLELT7EX6V'
+STELLAR_CHANNEL_SEEDS = os.environ['STELLAR_CHANNEL_SEEDS'].split(',')
```

Comment:
are you going to get it from KMS in the future?

Reply:
yes/ or some other wrapping tool will get it and pass it on